src/python/pants/backend/python/util_rules/lockfile_metadata.py

```diff
@@ -38,40 +53,53 @@ def from_lockfile(cls, lockfile: bytes) -> LockfileMetadata:
             elif in_metadata_block:
                 metadata_lines.append(line[2:])
 
+        error_suffix = "To resolve this error, you will need to regenerate the lockfile."
```

Comment:
"by running `./pants generate-lockfiles`"

Reply:
Should I mention `--resolve={tool_name}`?

Reply:
If possible, yes, that would be excellent! But I'm not sure how easy that will be to wire up.

Reply:
Thankfully, as easy as it was to wire in the other request you requested.